.github/workflows/Person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
```

Comment:
- [ Optional] Please `.github/workflows` directory is a place where you put the code you want Github to execute when you push something like Linters, also you can add a YAML file that runs your tests and so on, which doesn't make sense to put all your `.rb` files in it since it should contain only .yaml files.
![image](https://user-images.githubusercontent.com/74744787/216075968-8684b29f-9820-4d17-ae74-4c29d7bf1aae.png)